index.html

```diff
@@ -202,23 +209,20 @@ <h3>
                 </div>
             </div>
         </div>
-    </div>
-    <div class="about-page">
+    </section>
+    <section class="about-page">
         <div class="about-box">
-            <div class="about-heading">
-                <h2>
-                    About me
-                </h2>
-            </div>
-            <div class="about-description">
-                <p>I can help you build a product,<br>
-                    feature or website Look through<br>
-                    some of my work and experience! If<br>
-                    you like what you see and have a<br>
-                    project you need coded,<br>
-                    don’t hestiate to contact me.
-                </p>
-            </div>
+            <h2 class="about-heading">
+                About me
+            </h2>
+            <p class="about-description">
+                I can help you build a product,<br>
+                feature or website Look through<br>
+                some of my work and experience! If<br>
+                you like what you see and have a<br>
+                project you need coded,<br>
+                don’t hestiate to contact me.
+            </p>
```

Comment:
Kindly also fix the design of the about me section to match the Figma design as shown below 🤗
- _Make the section flexible to fill the screen_
- _Please add the background image to the section_

Your design:

![image](https://user-images.githubusercontent.com/73714615/187661744-b9fed3ab-937d-4dea-a75a-941df835bf90.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187661776-585c95bc-fc1c-453e-8768-9162c43b9fc0.png)